Source/Core/Core/Movie.h

```diff
@@ -101,7 +101,8 @@ struct DTMHeader
 	u8   bongos;
 	bool bSyncGPU;
 	bool bNetPlay;
-	u8   reserved[13];      // Padding for any new config options
+	bool bPAL60;
+	u8   reserved[12];      // Padding for any new config options
```

Comment:
Slightly off-topic. We are relying on sizeof(bool) to be 1, while in the c++ standard it is implementation defined.